packages/litexa/src/parser/state.coffee

```diff
@@ -307,7 +307,7 @@ class lib.State
       continue if name == '--default--'
       continue if intent.referenceIntent?
       unless intent.hasUtterances
-        unless context.skill.testDevice? and (name in extendedEventNames or name.includes('.')) # supported events have '.' in their names
+        unless ((name in extendedEventNames) or name.includes('Alexa.')) 
```

Comment:
Ya know, I'm not sure. It seemed too broad (and was incorrectly written, it actually wasn't skipping the message because the x in y clause is lower priority than x or y, so it was actually doing x in (y or z) instead of (x in z) or y!), so I narrowed it down to the events in the Alexa namespace for now, e.g.  the `Alexa.Presentation.HTML.Message` that made me look into it. Do you recall why we made it just '.' in the first place?

Reply:
I think the purpose here was to skip any custom intents that had no utterances as they'd be unreachable.

Builtin intents (`AMAZON.*`) and events (`Alexa.*`) both have periods, whereas custom intent names aren't permitted to have any.

But yeah, we don't want Litexa to skip unextended builtin intents here.

Reply:
Good point. Maybe I'll make it both `AMAZON.*` and `Alexa.*` as both have valid uses now, and leave the rest until we step on them.